backends/guest/common/read.c

```diff
@@ -186,30 +174,31 @@ int print_variables(const uint8_t *buffer, size_t buffer_size, const char *var_n
 				break;
 			case ST_SBAT:
 				printf("\tData: ");
-				print_raw((char *)cert, cert_size);
+				print_raw((char *)curr_esd.raw, esd_data_size);
+				break;
 			case ST_DELETE:
 				printf("\tDELETE-MSG: ");
-				print_raw((char *)cert, cert_size);
+				print_raw((char *)curr_esd.raw, esd_data_size);
+				break;
 			default:
-				prlog(PR_ERR, "ERROR: invalid signature type\n");
-				goto outside;
+				prlog(PR_ERR, "ERROR: invalid signature type = %d\n", sig_type);
+				break;
```

Comment:
99.9% sure was a bug, I don't know why we would want to fallthrough from `SBAT` into `DELETE` into `default`